psalm.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<psalm
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    cacheDirectory=".build/psalm"
+    errorBaseline="psalm-baseline.xml"
+    errorLevel="7"
```

Comment:
The value for this `errorLevel` attribute ranges between `8` (worst) and `1` (best).

The current value is the highest error level for which the code base currently passes the static code analysis with `vimeo/psalm`. I suggest that we increase it one level after another, until we get to `1`.